back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -108,6 +110,44 @@
         end
       end
 
+      if CitizenLab.ee?
+        example "List admin publications representing folders that contain project(s) with the specified areas;" do
+          a1 = create(:area)
+          a2 = create(:area)
+
+          p1 = @projects[0]
+          p1.areas << a1
+          p1.save!
+
+          do_request(areas: [a2.id], remove_childless_parents: true)
+          json_response = json_parse(response_body)
+
+          expect(json_response[:data].map { |d| d.dig(:relationships, :publication, :data, :id) }).to include @folder.id
+        end
+
+        example "Don't list admin publications representing folders that don't contain any project(s) with the specified areas;" do
+          a1 = create(:area)
+          a2 = create(:area)
+
+          p1 = @projects[0]
+          p1.areas << a1
+          p1.save!
+
+          p2 = @projects[1]
+          p2.areas << a1
+          p2.save!
+
+          p3 = @projects[2]
+          p3.areas << a1
+          p3.save!
```

Comment:
Do you really think it's worth the trouble of adding an extra area to the projects? If so, I wouldn't do it for the 3 projects then, to spare a few db queries and make the test faster.

Reply:
The reason I do it for all 3 projects in the folder is because currently we treat a project with no area selected as having all areas selected. My next PR in this Epic should remove that rule, and then I could take this test down to using only one project with a specified area.

Reply:
Alright, makes sense. May I suggest iteration then 🙂

Reply:
You may :monocle_face:

Reply:
Something like `@folder.projects.each {|p| p.update(areas: [a1]) }`.